src/scss/all-services.scss

```
@@ -0,0 +1,60 @@
+.all-services {
+    &__inner {
+        max-height: 490px;
+        padding-top: 100px;
+        background: #c4c4c4
+            linear-gradient(90deg, #504dc2 20.44%, #5f12d0 80.91%);
```

```suggestion
        background-image: linear-gradient(90deg, #504dc2 20.44%, #5f12d0 80.91%);
```

не зовсім розумію, для чого цей колір потрібен.
якщо він дійсно треба, тоді не змінюй нічого)